verification/curator-service/ui/src/components/Case.tsx

```diff
@@ -45,6 +45,7 @@ export interface Location {
     geometry: Geometry;
     name: string;
     place: string;
+    query?: string;
```

Comment:
Is this needed? The type description says `// Case definitions as returned by the /api/cases endpoint.` and I don't think query is ever returned from the backend, is it? If it's needed perhaps change the type description.

Reply:
Good call out, I'll change the type description. This is required to POST cases without a prior geocode. In the new case form, we use the geocode `suggest` endpoint, which takes care of completing the `location` object prior to use POSTing the new case. But for non-human-interactive usage, we're currently requiring a `query` field in `location` [in order to geocode](https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/verification/curator-service/api/src/controllers/cases.ts#L124).